index.html

```diff
@@ -287,6 +287,50 @@ <h5 class="lets-connect">LET’S CONNECT</h5>
           </li>
         </ul>
       </section>
+      <section class="contact-container">
+        <div class="contact-section">
+          <h2>Contact me</h2>
+          <p>
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it!
+          </p>
+          <form
+            class="contact-form-container"
+            action="https://formspree.io/f/{form_id}"
+            method="post"
+          >
```

Comment:
- [ ] Kindly make sure to create your account on [formspree service](https://formspree.io/html) which will provide a unique `id` to be used so the user can be able to send the message and even you the owner will be able to see the message from the user. At this moment the user can be able to the message because of the wrong unique `form_id`. Kindly fix that.

   - ### FORM BEHAVIOUR AFTER SUBMITTING
   
![image](https://user-images.githubusercontent.com/87197412/228469691-25505df2-50bc-4f48-b8bd-fdae2237a63b.png)